cro-clib/chain.h

```diff
@@ -100,12 +120,33 @@ CroResult cro_create_viewkey(CroHDWalletPtr wallet_ptr,
  */
 CroResult cro_destroy_address(CroAddressPtr addr);
 
+/**
+ * destroy fee
+ * # Safety
+ */
+CroResult cro_destroy_fee(CroFeePtr fee);
+
 /**
  * destroy bip44 hdwallet
  * # Safety
  */
 CroResult cro_destroy_hdwallet(CroHDWalletPtr hdwallet);
 
+/**
+ * destroy tx
+ * # Safety
+ */
+CroResult cro_destroy_tx(CroTxPtr tx);
+
+/**
+ * estimate fee
+ * # Safety
+ */
+CroResult cro_estimate_fee(CroFeePtr fee_ptr,
+                           uint32_t length,
+                           uint8_t *fee_output,
```

Comment:
What's the content of `fee_output` look like.

Reply:
fee is coin amount, so api gives back as "string",  i think u64 is better.
i will change it